### PR TITLE
Add dns_loopia

### PR DESCRIPTION
**Pull request by ivarmedi**
Loopia.se is one of the biggest registrars/web hosts in Sweden

### CODE REVIEW

**Comment by ivarmedi** [comment]
@Neilpang updated with documentation. Is there anything else I've missed from the dev guide?

**Comment by Neilpang** [comment]
Is Loopia.se dns firee ?
How can I create an account to test ?

**Comment by ivarmedi** [comment]
It's not free, sadly. But the API is documented here:
https://www.loopia.com/api/

File: README.md [hunk 1]
```diff
@@ -313,6 +313,7 @@ You don't have to do anything manually!
 1. zonomi.com DNS API
 1. DreamHost.com API
 1. DirectAdmin API
+1. Loopia API
```
Comment (Neilpang) [inline README.md]: can you please add the website url here ?

**Comment by ivarmedi** [comment]
@Neilpang sure thing, updated! I also used the _contains function instead of echo for the error checks

**Comment by Neilpang** [comment]
sorry, but please merge the latest code again.

**Pull request merged**